### PR TITLE
main thing is new custom hook for filtering data on homepage by date,…

### PR DESCRIPTION
**Pull request by stevenpmcfarlane**
… but also changes to quick selectbox

# Description
A few changes: big one is new custom hook for filtering data and then fixing the styling on the quick selectbox
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ X] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] `npm test`

# Checklist

- [ X] My code follows the style guidelines of this project
- [ X] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X ] There are no merge conflicts

### CODE REVIEW

**Review by robert-petersen — APPROVED** [review]
Nice screen share walk through on your changes!

File: src/hooks/useMapSearch.js [hunk 1]
```diff
@@ -69,9 +69,6 @@ export default function useMapSearch() {
         .filter(incident => moment(incident.date).isBetween(start, end))
         .map(incident => incident.id);
 
-      //const matches = dateIncidentList.filter(incident => moment(incident.date).isBetween(start, end)).map(incident => incident.id))
```
Comment (Brandonx1123) [inline src/hooks/useMapSearch.js]: deleting comments is good

File: src/components/Home/Map/Search.js [hunk 2]
```diff
@@ -109,17 +109,12 @@ export default function Search({ zoomOnCluster }) {
...
-            <option value="30">Within Past 30 Days</option>
-            <option value="10">Within Past 10 Days</option>
-          </select>
-        </label>
-      </div>
+
+      <Select onChange={onChange} value={quickSelect} name="quicksearch">
+        <Option value="">- Search Recent Reports -</Option>
+        <Option value="30">Within Past 30 Days</Option>
+        <Option value="10">Within Past 10 Days</Option>
```
Comment (Brandonx1123) [inline src/components/Home/Map/Search.js:116]: I like this feature with the custom hook, you confirmed with me on your local machine.

**Review by Brandonx1123 — APPROVED** [review]

**Review by john-chamberlin — APPROVED** [review]
I like it a lot

**Pull request merged**